include/cereal/external/base64.hpp

```diff
@@ -95,7 +95,7 @@ namespace cereal
         char_array_4[i++] = encoded_string[in_]; in_++;
         if (i ==4) {
           for (i = 0; i <4; i++)
-            char_array_4[i] = (unsigned char) chars.find( char_array_4[i] );
+            char_array_4[i] = static_cast<unsigned char>(chars.find( char_array_4[i] ));
```

Comment:
Is there a good reason for char_array* to be unsigned char, particularly when the string is a std::string.  alternatively, chars should probably be a fixed array, right?